modules/platform/nuxeo-platform-usermanager/src/test/java/org/nuxeo/ecm/platform/usermanager/io/NuxeoPrincipalJsonWriterTest.java

```diff
@@ -51,7 +51,7 @@ public void test() throws Exception {
         NuxeoPrincipal principal = userManager.getPrincipal("Administrator");
         JsonAssert json = jsonAssert(principal);
         json.isObject();
-        json.properties(6);
+        json.properties(7);
```

Comment:
Will probably be `6` if there's no partial content. It would be good to make sure that in this test class we have a test with and without partial content (and check `json.has("isPartial").isTrue()` in that case).